lib_nbgl/include/nbgl_use_case_v2.h

```diff
@@ -0,0 +1,162 @@
+
+/*
+ * Structure for more generic navigation
+ * the app specify a list of contents which can be of mixed types for example:
+ * - a CENTERED_INFO
+ * - then a TAG_VALUE_LIST
+ * - then a INFO_LONG_PRESS
+ */
+
+typedef void (*nbgl_contentCallback_t)(uint8_t contentIndex, nbgl_content_t *content);
+
+typedef struct {
+    const nbgl_content_t      *contentsList; ///< array of nbgl_content_t (nbContents items). If NULL, callback is used instead
+    nbgl_contentCallback_t     contentGetterCallback;  ///< function to call to retrieve a given content
+    uint8_t                    nbContents;  ///< number of contents
+} nbgl_genericContents_t;
```

Comment:
Alternative :
```
typedef struct {
    bool callback_call_needed;
    union {
        const nbgl_content_t      *contentsList; ///< array of nbgl_content_t (nbContents items). If NULL, callback is used instead
        nbgl_contentCallback_t     contentGetterCallback;  ///< function to call to retrieve a given content
    }
    uint8_t                    nbContents;  ///< number of contents
} nbgl_genericContents_t;
```
This better highlights the exclusive nature of content / callback